src/shared/components/modlog.tsx

```diff
@@ -578,9 +580,57 @@ export class Modlog extends Component<any, ModlogState> {
           </h5>
         ) : (
           <div>
-            <div class="table-responsive">
-              <table id="modlog_table" class="table table-sm table-hover">
-                <thead class="pointer">
+            <h5>
+              {this.state.communityName.isSome() && (
```

Comment:
Use a `this.state.communityName.match({some: communityName => ...` here.

Reply:
Okay, updated

Reply:
Okay, updated